### ISSUE TITLE
The accuracy of the test set remains unchanged.

### ISSUE THREAD

**Issue by xiaoxian7557** [TianyuanYang/MAML_for_Mechanical_Fault_Diagnosis#2]
Hello, the accuracy of my training gradually increases, but when it comes to testing, it stays almost constant from the first epoch. What could be the issue?

**Comment by TianyuanYang** [comment]
I think the improvement in accuracy with epoch increase during testing shouldn't be large, you could try changing the hyperparameters.

**Comment by xiaoxian7557** [comment]
Can I understand that the running result of the test.py file is used to further validate the reliability of the model based on the accuracy of the val obtained after running the train.py file?
